examples/python/crd/main.py

```diff
@@ -2,8 +2,8 @@
 from constructs import Construct
 from cdk8s import App, Chart
 
-from imports import jenkins
-from imports import clusterinstallation
+from imports.jenkins.io import jenkins
```

Comment:
Where did this change come from?

Reply:
This was originally noticed in this bug: https://github.com/awslabs/cdk8s/issues/273

The folder structure of the imports is like:

```
▁
/Users/campionfellin/Desktop/issue273/cdk8s/examples/python/crd/imports
├── autoscaling
|  └── internal
|     └── knative
|        └── dev
|           ├── metric
|           └── podautoscaler
├── caching
|  └── internal
|     └── knative
|        └── dev
|           └── image
├── jenkins
|  └── io
|     └── jenkins
|        ├── __init__.py
|        ├── _jsii
|        |  ├── __init__.py
|        |  └── jenkinsiojenkins@0.0.0.jsii.tgz
|        └── py.typed
├── k8s
|  ├── __init__.py
|  ├── _jsii
|  |  ├── __init__.py
|  |  └── k8s@0.0.0.jsii.tgz
|  └── py.typed
├── mattermost
|  └── com
|     └── clusterinstallation
|        ├── __init__.py
|        ├── _jsii
|        |  ├── __init__.py
|        |  └── mattermostcomclusterinstallation@0.0.0.jsii.tgz
|        └── py.typed
├── networking
|  └── internal
|     └── knative
|        └── dev
|           ├── certificate
|           ├── ingress
|           └── serverlessservice
├── sample_controller
|  └── k8s
|     └── io
|        └── foo
|           ├── __init__.py
|           ├── _jsii
|           └── py.typed
└── serving
   └── knative
      └── dev
         ├── configuration
         |  ├── __init__.py
         |  ├── _jsii
         |  └── py.typed
         ├── revision
         |  ├── __init__.py
         |  ├── _jsii
         |  └── py.typed
         ├── route
         |  ├── __init__.py
         |  ├── _jsii
         |  └── py.typed
         └── service
            ├── __init__.py
            ├── _jsii
            └── py.typed
```

So the import as it was, failed with:

```
    from imports import clusterinstallation
ImportError: cannot import name 'clusterinstallation' from 'imports' (unknown location)
```